File: tests/doma-sync-entities-from-database/fixtures/java-groovy-dsl-mysql/src/main/java/example/entity/Customer.java
package example.entity;

import org.seasar.doma.Column;
import org.seasar.doma.Entity;
import org.seasar.doma.Id;
import org.seasar.doma.Table;

/** Customers */
@Entity
@Table(catalog = "fixture_catalog", name = "customer")
public class Customer {

    /** Customer ID */
    @Id
    @Column(name = "customer_id")
    Long customerId;

    /** Returns the customerId. */
    public Long getCustomerId() {
        return customerId;
    }

    /** Sets the customerId. */
    public void setCustomerId(Long customerId) {
        this.customerId = customerId;
    }
}

File: tests/doma-sync-entities-from-database/fixtures/kotlin-groovy-dsl-mysql/src/main/kotlin/example/entity/Customer.kt
package example.entity

import org.seasar.doma.Column
import org.seasar.doma.Entity
import org.seasar.doma.Id
import org.seasar.doma.Metamodel
import org.seasar.doma.Table

/** Customers */
// HANDWRITTEN-BEGIN:audited-entity-annotations
@Audited
@Entity(metamodel = Metamodel())
// HANDWRITTEN-END:audited-entity-annotations
@Table(catalog = "fixture_catalog", name = "customer")
class Customer {

    /** Customer ID */
    @Id
    @Column(name = "customer_id")
    var customerId: Long = -1L

// HANDWRITTEN-BEGIN:audited-label
    /** Application-owned audited label. */
    fun auditedLabel(): String = "audited-$customerId"
// HANDWRITTEN-END:audited-label
}
